src/components/input/index.jsx

```diff
@@ -84,11 +85,17 @@ class Input extends React.Component {
         this.setValue(e.currentTarget.value)
     }
 
+    click(e) {
```

Comment:
thinking we probably dont need a click handler for inputs.. since that is essentially the focus function. Lemme know if you think otherwise